hardware_interface/include/hardware_interface/types/hardware_interface_return_values.hpp

```diff
@@ -17,9 +17,21 @@
 
 namespace hardware_interface
 {
-using hardware_interface_ret_t = int;
-static constexpr hardware_interface_ret_t HW_RET_OK = 0;
-static constexpr hardware_interface_ret_t HW_RET_ERROR = 1;
+enum class HardwareReturnType
```

Comment:
this should inherit from `std::uint8_t` or something

Reply:
roger roger